Dockerfile

```
@@ -82,9 +101,27 @@ RUN --mount=from=uv,source=/uv,target=/bin/uv \
     --mount=from=build_deps,source=${QE_APP_SRC},target=${QE_APP_SRC},rw \
     uv pip install --strict --system --compile-bytecode --cache-dir=${UV_CACHE_DIR} ${QE_APP_SRC}
 
+# TODO: this seems need to do twice
+# ENV UV_CONSTRAINT=${PIP_CONSTRAINT}
+# # Install the aiida-hyperqueue
+# # XXX: fix me after release aiida-hyperqueue
+# RUN --mount=from=uv,source=/uv,target=/bin/uv \
+#     --mount=from=build_deps,source=${UV_CACHE_DIR},target=${UV_CACHE_DIR},rw \
+#      git clone https://github.com/aiidateam/aiida-hyperqueue && \
+#      uv pip install --system --strict --compile-bytecode --cache-dir=${UV_CACHE_DIR} aiida-hyperqueue
```

```suggestion
    uv pip install --strict --system --compile-bytecode --cache-dir=${UV_CACHE_DIR} \
     ${QE_APP_SRC} "aiida-hyperqueue @ git+https://github.com/aiidateam/aiida-hyperqueue" 
```

I think this should do the trick